_posts/meat/2020-03-29-10lb-pork-shoulder.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: Cook a 10 lb Pork Shoulder
```

Comment:
Possible alternative title: `Roast Pork Shoulder`, to be a food noun.

Reply:
ugh

Reply:
is this a blocking comment